### ISSUE TITLE
Test incident 2

### ISSUE THREAD

**Issue by cmeury** [bananafon/blue#4]
Test

**Comment by jeremycalles** [comment]
It is still not working.

**Comment by cmeury** [comment]
What do you mean, it's not working :)

**Comment by jeremycalles** [comment]
It is a bug, so it is not working. :+1:

**Comment by cmeury** [comment]
But what is not working, the synchronization between GitHub and Zube seems to work fine?

**Comment by jeremycalles** [comment]
This is a test, so I had to write something. And I did write "This is not working" but this is working fine actually.
Next time I will put "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
